utils/sanity.client.ts

```diff
@@ -18,18 +18,40 @@ const sanityClient = (token?: string) => {
   return projectId ? createClientCall : null;
 };
 
-export async function getCalculatorPageBySlug({
-  slug,
-  token,
-}: {
+interface CalcPageBySlugParams {
   slug: string;
   token?: string;
-}) {
-  return sanityClient(token)?.fetch(calculatorPagesBySlugQuery, { slug });
 }
 
-export async function getCalculatorPagePaths(): Promise<string[] | undefined> {
-  return sanityClient()?.fetch(calculatorPagePaths);
+// `calculatorInfoPage` is the name of the Sanity MISDEMEANOR page, where the visual "title" for
```

Comment:
were these comments meant to be separated?

Reply:
😒 there is an eslint rule that lines can't be > 100 chars

Reply:
I updated it to try and make it less ugly.